index.html

```diff
@@ -0,0 +1,156 @@
+        <h1 class="main-title"><q>Irreechaa  Love,Peace and Unity</q></h1>
+        <div class="hero-title">CELEBRATING IRREECHAA FESTIVAL 2023</div>
+        <div class="description-text">
+          A joyful celebration believing in value of openness and sharing. Creating a positive change with people from every corner of the countries is taking place in October in Ethiopian capital Finfinnee.
+        </div>
+        <h2 class="program-time">2023,10,15(saturday)-16(sunday)</h2>
+        <p class="program-place">
+          @Finfinnee, Hora Finfinnee and Bishooftuu, Hora Arsadii          
+        </p>
+      </section>
+      <section class="main-program">
+        <h3 class="main-program-title">Main Program</h3>
+        <div class="program-container">
+          <div class="content">
+            <span class="icon">
+              <img src="./assets/images/assembly.png" alt="Lecture" />
+            </span>
+            <h4 class="title">Lecture</h4>
+            <p>
+              Have to share your thoughts and opinions with experts on each Irreechaa celebration festival disciplines
+            </p>
+          </div>
+          <div class="content">
+            <span class="icon">
+              <img src="./assets/images/exhibition.png" alt="exhibition" />
+            </span>
+            <h4 class="title">Exibition</h4>
+            <p>
+              Have to share your thoughts and opinions with experts on each Irreechaa celebration festival disciplines
+            </p>
+          </div>
+          <div class="content">
+            <span class="icon">
+              <img src="./assets/images/forum.png" alt="forum" />
+            </span>
+            <h4 class="title">Forum</h4>
+            <p>
+              Have to share your thoughts and opinions with experts on each Irreechaa celebration festival disciplines
+            </p>
+          </div>
+          <div class="content">
+            <span class="icon">
+              <img src="./assets/images/workshop.png" alt="workshop" />
+            </span>
+            <h4 class="title">Workshop</h4>
+            <p>
+              Have to share your thoughts and opinions with experts on each Irreechaa celebration festival disciplines
+            </p>
+          </div>
+          <div class="content">
+            <span class="icon">
+              <img src="./assets/images/ignite.png" alt="ignite" />
+            </span>
+            <h4 class="title">Irreechaa Ignite</h4>
+            <p>
+              Have to share your thoughts and opinions with experts on each Irreechaa celebration festival disciplines
+            </p>
+          </div>
+        </div>
+        <button class="join-btn">Join Erriecha Festival 2023</button>
+        <div class="see-program">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+    </main>
+    <section class="featured-speaker" id="dynamicPage">
+      
+    </section>
```

Comment:
- [ ] As suggested by the previous reviewer, here in this section kindly consider fixing following:
  -  your images and background design vectors are breaking the and messing the design on most of the screen sizes. that's because they are flex items. I think here you can apply fixed widths to the speaker images and the design vector images behind them so that they both remain same at all screen sizes.
  - the description text color is not like in the Behance design so try fixing it. 
  - Also the font sizes and font weights in all three name, profession and description are not similar to the Behance design so try fixing it.
your design
![df](https://user-images.githubusercontent.com/96683843/226132021-5cc8c1ef-85c8-403c-8d01-009270939128.jpg)
![as](https://user-images.githubusercontent.com/96683843/226132070-7b3e4932-496f-4892-a298-ff1196df42b0.jpg)
Behance design
![po](https://user-images.githubusercontent.com/96683843/226132167-a276aa43-b24b-4ceb-a65c-7e193ba85393.jpg)